### PR TITLE
Debugged why png files are not sent

### PR DESCRIPTION
**Pull request by nnk03**
The problem was in the way, files were sent `File.ReadAllLines` were used which works only for text files, so for png and other types of files, it fails. Hence, now using byte array to send the files

### CODE REVIEW

File: FileCloner/Models/Constants.cs [hunk 1]
```diff
@@ -47,6 +47,9 @@ public class Constants
         public const string broadcast = "BroadCast";
         public static string IPAddress = GetIP();
 
+        // Size of FileChunk to be sent over network
+        public static int FileChunkSize = 4096;
```
Comment (dshreddy) [inline FileCloner/Models/Constants.cs:51]: Any particular reason why the chunk size can't be larger or smaller than 4096 ?

**Review by dshreddy — COMMENTED** [review]
LGTM, let me know after testing it using 3 systems

File: FileCloner/Models/Constants.cs (hunk 1 above)
Comment (nnk03) [reply FileCloner/Models/Constants.cs:51]: No particular reason, 2 KB and 3 KB felt too small and anything more than 4 KB felt too large.

Comment (nnk03) [reply FileCloner/Models/Constants.cs:51]: 😅

Comment (dshreddy) [reply FileCloner/Models/Constants.cs:51]: Alright but just want to confirm, it doesn't affect the net amount of time required to send / receive a file right ?

**Comment by dshreddy** [comment]
> The problem was in the way, files were sent `File.ReadAllLines` were used which works only for text files, so for png and other types of files, it fails. Hence, now using byte array to send the files

File: FileCloner/Models/Constants.cs (hunk 1 above)
Comment (nnk03) [reply FileCloner/Models/Constants.cs:51]: Yes, rectified by increasing the chunk size in the next PR

**Pull request merged**